src/gui/clips/AutomationClipView.cpp

```diff
@@ -390,6 +394,11 @@ void AutomationClipView::paintEvent( QPaintEvent * )
 		p.drawPixmap( spacing, height() - ( size + spacing ),
 			embed::getIconPixmap( "muted", size, size ) );
 	}
+	
+	if (m_marker)
+	{
+		p.drawLine(m_markerPos, rect().bottom(), m_markerPos, rect().top());
+	}
```

Comment:
What does this do? I'm not sure why it's in the double click event

Reply:
It draws a vertical line at the split position as you hold down the mouse on knife mode, so that you can see where it will split before actually splitting it. It's also in `paintEvent()`, not `mouseDoubleClickEvent()`.

Reply:
Ah my bad!